website/docs/r/efs_file_system_backup_policy.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "EFS"
+layout: "aws"
+page_title: "AWS: aws_efs_file_system_backup_policy"
+description: |-
+  Provides an Elastic File System (EFS) File System Backup Policy resource.
+---
+
+# Resource: aws_efs_file_system_backup_policy
+
+Provides an Elastic File System (EFS) File System Backup Policy resource.
+
+## Example Usage
+
+```hcl
+resource "aws_efs_file_system" "fs" {
+  creation_token = "my-product"
+}
+
+resource "aws_efs_file_system_backup_policy" "policy" {
+  file_system_id = aws_efs_file_system.fs.id
+
+  backup_policy {
+    status = "ENABLED"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `file_system_id` - (Required) The ID of the EFS file system.
+* `backup_policy` - (Required) A file system backup_policy object (documented below).
+
+### Backup Policy Arguments
+For **backup_policy** the following attributes are supported:
+
+* `status` - (Required) A status of the backup policy. Valid values: `Enabled`.
```

Comment:
Is there any value here to supporting `Disabled` as well, or would that be the same as an absence of the `efs_file_system_backup_policy` resource?

Reply:
I went ahead and pushed a change to allow `DISABLED`.